index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta content='Linkbrary' property='og:title'/>
+  <meta content='//www.wjyweekly.netlify.app/' property='og:url'/>
+  <meta content='세상의 모든 정보를 쉽게 저장하고 관리해 보세요' property='og:description'/>
+  <meta content='./img/page.png' property='og:image'/>
+  <title>Linkbrary</title>
+  <link rel="stylesheet" href="css/indexStyle.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <link
+  href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&family=Poppins:wght@400;700&display=swap"
+  rel="stylesheet"
+>
+</head>
+<body>
+  <!-- 네이게이션 바 -->
+  <nav class="navigationBar">
+    <div class="nav-container">
+      <a class="btn-home" href="./index.html">Linkbrary</a> <a class="btn btn-login" href="./login.html">로그인</a>
+    </div>
+  </nav>  
+  <!-- 헤더 -->
+  <header class="header">
+    <div class="header-container">
+      <h2 class ="header-contents-txt">
+        <span class="text-gradient-purple-to-pink">세상의 모든 정보</span>를<br>
+        쉽게 저장하고<br class="only-tablet"> 관리해보세요
+      </h2>
+      <a class="btn add-link" href="#">링크 추가하기</a>
+      <img src="./img/hero.png" class="hero-img">
+    </div>
+  </header>
```

Comment:
@wjy308 

header는 특정 콘텐츠의 헤딩을 포함하는 요소를 나타냅니다. 
이 섹션의 겨웅, 헤딩을 포함하기는 하지만 홈페이지에 이런 요소들을 보통 Hero 섹션이라고 표현하죠.
따라서 header라고 하기보단, hero라는 섹션으로 두는게 더 좋아보입니다. 

따라서 개선해본다면 + 공통 css를 활용해본다면 다음과 같이 해볼 수 있겠네요

```suggestion
<section class="hero">
  <div class="container hero-container">
    <div class="hero-container__text-box">
      <h2 class="hero-container__text-box__title text-gradient-purple-to-pink">
      ...
    </ div>
    <div class="hero-container__img-box">
    </ div>
```